.gitignore

```diff
@@ -289,3 +289,5 @@ $RECYCLE.BIN/
 *.lnk
 
 # End of https://www.gitignore.io/api/node,macos,linux,windows,firebase,reactnative
+
+.vscode
```

Comment:
go to https://www.gitignore.io/ and type in `Visual Studio` then copy the entire block into the gitignore

Reply:
ya super handy. always use it for new repos